src/main.rs

```diff
@@ -904,10 +904,8 @@ fn bisect_nightlies(cfg: &Config, client: &Client) -> anyhow::Result<BisectionRe
                         "the start of the range ({}) must not reproduce the regression",
                         t
                     );
-                } else {
-                    last_failure = nightly_date;
                 }
-
+                last_failure = nightly_date;
```

Comment:
this changes behaviour

Reply:
I don't think it does,  if any conditional branch is activated, this line is not reached. Was caught by clippy.